doc/user_guide/tools.rst

```diff
@@ -62,12 +64,12 @@ information (including calibration) can be accessed (and modified) in the
 Transforming between signal subclasses
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The different subclasses are characterized by three
+The different subclasses are characterized by four
 :py:attr:`~.signal.BaseSignal.metadata` attributes (see the table below):
 
 `record_by`
     Can be "spectrum", "image" or "", the latter meaning undefined and describes
-    the way the data is arranged in memory. It is possible to transform any
+    the way the data is arranged in memory. It is possible to transform any non-complex
```

Comment:
Good point. Why not providing a `to_signal1D` method for `ComplexSignal2D`. A `to_signal2D` method would probably justify adding a `ComplexSignal1D` class. The current `to_signal1D` and `to_signal2D` should operate well on complex data. Maybe the could be place in new `CommonSignal1D` and `CommonSignal2D` classes?